### DISCUSSION TITLE
Error when generating a container on docker with prisma, incorrectly pinned it to linux-musl

### DISCUSSION THREAD

**Discussion by HenriqueAurelio** [prisma/prisma#13218] [Q&A] [answered]
My error when i use docker-compose up says that i have to specifically put linux on binaryTargets but he is already there

![image](https://user-images.githubusercontent.com/31359644/167139390-9dd2d029-aa40-4888-9431-585e1a9f922b.png)
My Dockerfile
![image](https://user-images.githubusercontent.com/31359644/167139528-f33a870d-6d45-49ba-87aa-de3e03c523d8.png)
My docker-compose file
![image](https://user-images.githubusercontent.com/31359644/167139579-6fd92d0a-df41-4e4b-8768-cbff1dc03ed4.png)
My schema file
![image](https://user-images.githubusercontent.com/31359644/167140120-16e89644-f13e-4f5c-a031-9107ec456db9.png)

**Comment by nurul3101** [comment, marked as answer]
Hey @HenriqueAurelio 👋 
Can you try upgrading your `@prisma/client` to version `3.13.0` and check?

We added support for OpenSSL 3.0 from version [3.13.0](https://github.com/prisma/prisma/releases/tag/3.13.0). I am assuming that maybe your OS could default to OpenSSL 3.0 which might have caused this error.

**Reply by HenriqueAurelio** [reply]
Thanks a lot it worked :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
